libs/ui-watt/src/lib/foundations/color/storybook/color.component.scss

```diff
@@ -0,0 +1,37 @@
+@use 'sass:map';
+@use './../colors.import' as colors;
+
+:host {
+    background: colors.getColor('grey-300');
+    display: block;
+    width: 100%;
+    padding: 1rem;
+    margin: -1rem;
+}
+
+mat-card {
+    margin-bottom: 8px;
+}
+
+.color-grid {
+    display: flex;
+    flex-wrap: wrap;
+    max-width: 688px;
+}
+
+figure {
+    margin: 10px;
+}
+
+figcaption {
+    text-transform: uppercase;
+}
```

Comment:
Suggestion: Use the `.watt-label` on the element. It will uppercase the text automatically.

Reply:
Totally agree, will fix asap